src/net/peacefulcraft/cowbot/CowBot.java

```diff
@@ -46,12 +46,14 @@ public CowBot() {
 
   public void onEnable() {
     config = new Configuration();
-    if (config.getBotToken().length() > 0) {
+    if (config.getDatabaseName().length() > 0) {
```

Comment:
I changed the condition here to allow building and testing of CowBot features without having a DB set up. If the DB fields are empty up in the config, or they're set up but not actually linked to a DB, the HikariManager constructor will throw and you can't test the non-DB related features. The DB related functionality all checks if the dbPool is null anyway, so I figured this wasn't dangerous to change permanently, and would reduce one more error future contributors have to grapple with.

You used to implicitly need a DB name for the block on lines 49-54, but now it's explicitly required. If a DB name isn't specified, you can still run CowBot so long as you don't attempt to DB-related features. That sums up the change. I can put this in a separate PR, or drop it altogether if you want, but it's a very nice QoL change for local development.